packages/app/src/debug/DebugContainer.vue

```diff
@@ -1,16 +1,30 @@
 <template>
-  <div>
+  <div class="h-full">
     <DebugError
       v-if="showError"
     />
     <div
-      v-else-if="loginConnectStore.user.isLoggedIn && loginConnectStore.project.isProjectConnected && run"
+      v-else-if="loginConnectStore.user.isLoggedIn && loginConnectStore.project.isProjectConnected && run && run.status"
```

Comment:
Should we name this, something like `hasCloudRun`?

Reply:
I'd like to keep it this way because when I move this into a computed value, the types in the template get all messed up. 

I guess because it doesn't realize that we're checking for the existence of `run` and `run.status` in the computed function? Because it still thinks that `run` could be null inside of the `v-else-if` block even though at that point we've determined that `run` and `run.status` are not null 😕

Reply:
We could at least combine the last two conditions using a null-safe traversal `&& run?.status`, right?